patches/server/1052-Add-method-to-reset-Lodestone-compasses-back-to-norm.patch

```diff
@@ -0,0 +1,55 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: DerEchtePilz <81232921+DerEchtePilz@users.noreply.github.com>
+Date: Tue, 20 Aug 2024 12:38:55 +0200
+Subject: [PATCH] Add method to reset Lodestone compasses back to normal ones
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/inventory/CraftMetaCompass.java b/src/main/java/org/bukkit/craftbukkit/inventory/CraftMetaCompass.java
+index 6362df65424e53098701b8d54c74b5905648b78a..95f41c1315d6cca8f35eb915fa4d6a9f510a4633 100644
+--- a/src/main/java/org/bukkit/craftbukkit/inventory/CraftMetaCompass.java
++++ b/src/main/java/org/bukkit/craftbukkit/inventory/CraftMetaCompass.java
+@@ -36,6 +36,7 @@ public class CraftMetaCompass extends CraftMetaItem implements CompassMeta {
+     private int lodestoneY;
+     private int lodestoneZ;
+     private Boolean tracked = null; // Paper - tri-state
++    private boolean shouldResetLodestone = true; // Paper - Add method to reset Lodestone compasses back to normal ones
```

Comment:
How is adding this field different from just setting the tracked Boolean to null. That's why we made the `tracked` field a "tri-state" with a nullable boolean.

Reply:
I actually did not think about that, but it definitely makes sense.
I did change that now.